spec/rails_helper.rb

```diff
@@ -9,9 +9,25 @@
 
 require 'capybara-screenshot/rspec'
 
-# Poltergeist with PhantomJS
-require 'capybara/poltergeist'
-Capybara.javascript_driver = :poltergeist
+# Headless Chrome
+require 'selenium/webdriver'
+
+Capybara.register_driver :chrome do |app|
+  Capybara::Selenium::Driver.new(app, browser: :chrome)
+end
+
+Capybara.register_driver :selenium do |app|
```

Comment:
Doesn't change anything functionally, and maybe misunderstanding this, but should this be `:headless_chrome` or something instead of `:selenium`? It looks like it's "chrome driving selenium".